apps/web-portal-client/lib/components/nav-bar/LoginButton.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+export function LoginButton(props) {
+  const {label, onClick} = props
+
+  return (
+    <div>
+      <button
+        type="button"
+        onClick={onClick}
+        className="bg-gray-800 inline-flex items-center justify-center p-2 rounded-md text-gray-400 hover:text-white hover:bg-gray-700 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-offset-gray-800 focus:ring-white"
```

Comment:
Extract to separate css file, take a look on 
https://github.com/zorko-io/zorko-designer/blob/master/src/components/Button.tsx

Reply:
move css in `LoginButton.css`